pyxem/utils/atomic_scattering_params.py

```diff
@@ -0,0 +1,103 @@
+# Electron scattering factors from International Tables Vol. C, table 4.3.2.3
```

Comment:
These are now in diffsims - can you import what you need from there instead?

Reply:
And the same for other scattering parameterizations...

Reply:
Okay, will change code